packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -224,6 +224,22 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for Policy Guidelines
+    ///
+
+    // RPC Code: RPC-Policy-V1-03
+    AvoidAdditionalProperties: {
+      description: "The Additional Properties is not allowed except for tags.",
```

Comment:
>The Additional Properties is not allowed except for tags. [](http://example.com/codeflow?start=20&length=57)

The use of additionalProperties is not allowed except for user defined tags on tracked resources.

Reply:
The field of type additionalProperties is not allowed except for properties.tags used in tracked resources.